### ISSUE TITLE
Papermerge - Can't change OCR languages

### ISSUE THREAD

**Issue by samuelsson86** [alexbelgium/hassio-addons#263]
Hi,

I tried changing the ocr languages in Papermerge to Norwegian and English with Norwegian being the default language.

In the UI I have this:
```
ocrlang: nor,eng
```

And in the config file this:

```
OCR_DEFAULT_LANGUAGE` = "nor"

OCRLANG = {
  "nor" : "Norwegian",
  "eng" : "English"
```

This is the log when starting the addon:
```
11:14:59] INFO: Configuring 2 languages
[11:14:59] INFO: OCRLANG variable is set, processing the language packages
[11:14:59] INFO: Writing new configuration
[11:15:02] INFO: Processing language nor
[11:15:03] INFO: Installing tesseract-ocr-nor
[11:15:07] INFO: nor identified as Norwegian
[11:15:07] INFO: Setting default language to nor
[11:15:07] INFO: ... nor installed
[11:15:07] INFO: Processing language eng
[11:15:08] INFO: Installing tesseract-ocr-eng
[11:15:10] INFO: eng identified as English
[11:15:10] INFO: ... eng installed
```

But I can still only choose between German and English in Papermerge. German not even being a chosen language. 
Is there something I'm missing?

Papermerge Docs say something about changing the settings in both main app and worker but I have no idea how to do that with this addon.

**Comment by alexbelgium** [comment]
Hi, it is now solved in version v2.0.1-ls40. Could you please test? Thanks

**Comment by samuelsson86** [comment]
Hi,

Thanks for your quick reply. Amazing.
It works now.
